### PR TITLE
fix(Strategy): cancel exit orders

### PR DESCRIPTION
**Pull request by yakir4123**
in case i want to cancel both of the exit orders without setting new exit orders I would like to do something like this:

```
self.stop_loss = []
self.take_profit = [] 
```
but it will raise an error of `'stop-loss and take-profit should not be exactly the same. Just use either one of them and it will do.'`

### CODE REVIEW

**Comment by saleh-mir** [comment]
Thanks man, another good one.
